crates/re_space_view_time_series/src/util.rs

```diff
@@ -150,7 +150,14 @@ pub fn apply_aggregation(
     // So it can be displayed in the UI by the SpaceViewClass.
     let num_points_before = points.len() as f64;
 
-    let points = if aggregation_duration > 2.0 {
+    // If the user logged multiples scalars per time stamp, we should aggregate them,
+    // no matter what the aggregation duration (=zoom level) is.
+    let multiple_values_per_time_stamp = || points.windows(2).any(|w| w[0].time == w[1].time);
+
+    let should_aggregate = aggregator != TimeSeriesAggregator::Off
+        && (2.0 <= aggregation_duration || multiple_values_per_time_stamp());
```

Comment:
I believe we could handle this special case in a much cheaper way if we do it using an accumulator while we are computing the `PlotPoint`s to begin with (no search for duplicates, no allocating unnecessary `PlotPoint`s, etc).

It's fine as a quick fix for the time being but we should open an issue then.

Reply:
It will be a lot of duplicated code though, for averaging the plot point properties

Reply:
> Can't the same argument be made for the whole of the aggregation; that it can be run in a streaming fashion without first allocating all the plot points?

I'm not sure it can off the top of my head, but at the very least it would be much much more tricky because of fractional pixels, inter-timestamp aggregation, edge cases that require peeking your neighbours, etc.

While this specific case is just constrained to a single timestamp with no shenanigans.

Reply:
It is also not worth optimizing this right now; the aggregation is dwarfed by the query time:

![image](https://github.com/rerun-io/rerun/assets/1148717/046a0c56-f986-4362-98fc-4d92050badaf)